tests/darwin/client_test.py

```diff
@@ -1,88 +1,11 @@
+import logging
 from pathlib import Path
-from typing import Any, Dict
 
-import pytest
 import responses
 
 from darwin.client import Client
 from darwin.config import Config
 from darwin.datatypes import Feature
-from darwin.exceptions import InsufficientStorage, Unauthorized
```

Comment:
We are removing some tests, and we're not adding any for the new functions we're introducing. Are we ok with this?

Reply:
Removed 2 outdated tests. Added more than that. Obviously every single function should be tested. But if I do that now it might be overkill for a single PR.